stories/wizard/wizard.stories.js

```diff
@@ -0,0 +1,868 @@
+The Wizard consists of the walkthrough screen, where the form sections are revealed subsequently after completing, and the summary page, where the form is displayed in read only mode for assessment and final submission.
+
+
+## Usage
+
+**Use the wizard if:**
+
+- the user has to accomplish a long or unfamiliar task
+- the flow consist of a minimum of 3 and a maximum of 8 steps
+
+**Do not use the wizard if:**
+
+- a task has only 2 steps
+- a task has more than 8 steps
+- the format of the task is familiar to the user
+
+## Layout
+The wizard can be used both in a full-screen layout as well as in the flexible column layout. There is no navigation from the wizard to a next page. After completion or cancellation, the user is always navigated to the page the wizard was triggered from.
+
+## Responsive paddings
+
+These modifier classes will add horizontal paddings to the content and can be applied on the <code>fd-wizard\\_\\_progress-bar</code> level and/or on the <code>fd-wizard\\_\\_content</code> level.
+
+
+|  &nbsp;&nbsp;**rem** &nbsp;&nbsp; |  &nbsp;&nbsp;**min-width** &nbsp;&nbsp; |  &nbsp;&nbsp;**max width** &nbsp;&nbsp; |  &nbsp;&nbsp;**modifier class** &nbsp;&nbsp; |
+| :----: | :----------: | :----------: | :----------------------------------: |
+| 1rem | _n/a_ | 599px | <code>fd-wizard\\_\\_progress-bar--s</code> / <code>fd-wizard\\_\\_content--s</code> |
+| 2rem | 600px | 1023px | <code>fd-wizard\\_\\_progress-bar--m</code> / <code>fd-wizard\\_\\_content--m</code> |
+| 2rem | 1024px | 1439px | <code>fd-wizard\\_\\_progress-bar--l</code> / <code>fd-wizard\\_\\_content--l</code> |
+| 3rem | 1440px | _n/a_ | <code>fd-wizard\\_\\_progress-bar--xl</code> / <code>fd-wizard\\_\\_content--xl</code> |
+
+
+## Step types
+
+Use the following modifier classes to change the type of the wizard step:
+
+
+| **Type** | **Modifier class** |
+| :---- | :--------------: |
+| completed     |  &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_step--completed</code>     |
+| current       |  &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_step--current</code>       |
+| upcoming      |  &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_step--upcoming</code>      |
+| no-label      |  &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_step--no-label</code>      |
+| stacked       |  &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_step--stacked</code>       |
+| stacked-top   |  &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_step--stacked-top</code>   |
+| active        |  &nbsp;&nbsp; &nbsp;&nbsp; <code>fd-wizard\\_\\_step--active</code>        |
+
+## Connector types
+
+When connecting Inactive steps, or Active and Inactive steps, use the default <code>fd-wizard\\_\\_connector</code> class. For connecting Active steps the <code>fd-wizard\\_\\_connector--active</code> modifier class should be applied together with <code>fd-wizard\\_\\_connector</code>. For branching step connector use <code>fd-wizard\\_\\_connector--branching</code> modifier class.
```

Comment:
Just want to make sure here: Inactive steps are the greyed out steps that have yet to be selected right?

Reply:
There are multiple connector types that can be displayed depending on the steps involved.

| Connector type | Connection | Modifier/Class |
| :--------------- | :----------- | :------------- |
| Default | Inactive step (or Active step) to Inactive step | <code>fd-wizard\\_\\_connector</code> |
| Active | Active step to Active step | <code>fd-wizard\\_\\_connector--active</code> |
| Branching | Branching step to Inactive step | <code>fd-wizard\\_\\_connector--branching</code> |

Reply:
Does this make sense? @InnaAtanasova